releases/mainnet/1.9.5/index.html

```diff
@@ -0,0 +1,27 @@
+<html>
+	<head>
+		<title>Mainnet Release 1.9.5</title>
+	</head>
+	<body>
+		<div>
+			<h2>
+				<strong>Mainnet Release 1.9.5</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/mainnet/1.9.5/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmWXAVJVtp42HxPZrPGm6aTGYK4vXoW8Q1uuJVa31A82Md">Contract addresses (IPFS hosted)</a></li>
```

Comment:
This doesn't open for me.

Reply:
Sometimes it takes a few seconds to find it on the network. It's working for me.

Reply:
Yeah working here as well.